### PR TITLE
Update values.yaml

### PR DESCRIPTION
**Pull request by rish691**

### CODE REVIEW

File: helm/values.yaml [hunk 1]
```diff
@@ -72,6 +72,10 @@ logConfig:
       enabled: false
 
 config-bootstrapper:
-  job:
-    prefix: attribute
-  configurationCommands: {}
\ No newline at end of file
+  jobs:
```
Comment (tim-mwangi) [inline helm/values.yaml:75]: Do you need to update the config bootstrapper dependency version somewhere?

Comment (rish691) [reply helm/values.yaml:75]: Yes, this pr and https://github.com/hypertrace/entity-service/pull/77 needs to be updated with latest config-bootstrapper chart version once 
https://github.com/hypertrace/config-bootstrapper/pull/33 is merged

Comment (tim-mwangi) [reply helm/values.yaml:75]: ok.

**Comment by rish691** [comment]
No longer needed

**Pull request closed without merging**